apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -73,11 +74,24 @@ class LessonOverview extends Component {
     // from redux
     announcements: PropTypes.arrayOf(announcementShape),
     viewAs: PropTypes.oneOf(Object.values(ViewType)).isRequired,
-    isSignedIn: PropTypes.bool.isRequired
+    isSignedIn: PropTypes.bool.isRequired,
+    isVerifiedTeacher: PropTypes.bool.isRequired,
+    hasVerifiedResources: PropTypes.bool.isRequired
   };
 
   render() {
-    const {lesson, announcements, isSignedIn, viewAs} = this.props;
+    const {
+      lesson,
+      announcements,
+      isSignedIn,
+      viewAs,
+      isVerifiedTeacher,
+      hasVerifiedResources
+    } = this.props;
+
+    const displayVerifiedResources =
```

Comment:
nit: `displayVerifiedResources` sounds like this is a flag to display the resources not the notification. Could it be `displayVerifiedResourcesNotification` or something instead?

Reply:
Done!